frontend/src/components/form/data_entry/polling_station_choice/util.ts

```diff
@@ -0,0 +1,56 @@
+import { DataEntryStatusName, ElectionStatusResponseEntry, LoginResponse, PollingStation } from "@/api";
+
+export enum PollingStationUserStatus {
+  MISSING = "MISSING",
+  IN_PROGRESS_CURRENT_USER = "IN_PROGRESS_CURRENT_USER",
+  IN_PROGRESS_OTHER_USER = "IN_PROGRESS_OTHER_USER",
+  FINISHED = "FINISHED",
+  UNKNOWN = "UNKNOWN",
+}
```

Comment:
```suggestion
export enum PollingStationUserStatus {
  MISSING,
  IN_PROGRESS_CURRENT_USER,
  IN_PROGRESS_OTHER_USER,
  FINISHED,
  UNKNOWN,
}
```
There is also no reason for the enum members to be upper case.

Reply:
when you dont set a value the debug messages are a bit less easy to read (ints) so thats why I added it.
as for upper case. it's a convention I'm used to, but i'll make it conform the rest of the app.

Reply:
renamed